code/modules/admin/admin_verbs.dm

```diff
@@ -202,6 +203,48 @@ var/list/admin_verbs_mentor = list(
 		if(holder.rights & R_SPAWN)			verbs += admin_verbs_spawn
 		if(holder.rights & R_MOD)			verbs += admin_verbs_mod
 		if(holder.rights & R_MENTOR)		verbs += admin_verbs_mentor
+		
+/client/proc/remove_admin_verbs()
+	verbs.Remove(
+		admin_verbs_default,
+		/client/proc/togglebuildmodeself,
+		admin_verbs_admin,
+		admin_verbs_ban,
+		admin_verbs_event,
+		admin_verbs_server,
+		admin_verbs_debug,
+		admin_verbs_possess,
+		admin_verbs_permissions,
+		/client/proc/stealth,
+		admin_verbs_rejuv,
+		admin_verbs_sounds,
+		admin_verbs_spawn,
+		admin_verbs_mod,
+		admin_verbs_mentor,
+		admin_verbs_show_debug_verbs,
+		/client/proc/readmin,
+	)
+
+/client/proc/hide_verbs()
+	set name = "Adminverbs - Hide All"

```

Comment:
Um, this was removed a long time ago. No one used it. I am almost sure that Ponies and Neca told me not to readd it, when I inquired about it.